docs/how-to-guides/shared-libraries.rst

```diff
@@ -64,11 +64,16 @@ A shared library produced by ``library()`` or ``shared_library()`` built like th
 
 is installed to ``libdir`` by default. If the only reason the shared library exists
 is to be used inside the Python package being built, then it is best to modify
-the install location to be within the Python package itself:
+the install location, via the `install_dir` argument, to be within the Python package itself:
 
-.. code-block:: python
+.. code-block:: meson
 
-   install_path: py.get_install_dir() / 'mypkg/subdir'
+    example_lib = shared_library(
+        'example',
+        'examplelib.c',
+        install: true,
+        install_dir: py.get_install_dir() / 'mypkg/subdir',
```

Comment:
I've just realized that this example and other in this file uses the Meson's `python` module imported as `py`. However, the example just below uses `py3`. While at this, can you please fix this inconsistency too (in a separate commit maybe)?